Makefile

```diff
@@ -242,8 +242,12 @@ local-build: ## Build all container images necessary to run the whole operator
 	$(CONTAINER_TOOL) build -v $(GO_CONTAINER_CACHE):/go:z -f Dockerfile.mrvlVSP.rhel -t $(MARVELL_VSP_IMAGE)
 	$(CONTAINER_TOOL) build -v $(GO_CONTAINER_CACHE):/go:z -f Dockerfile.IntelVSP.rhel -t $(INTEL_VSP_IMAGE)
 
+.PHONE: prepare-multi-arch
+prepare-multi-arch:
+	test -f /proc/sys/fs/binfmt_misc/qemu-aarch64 || sudo podman run --rm --privileged quay.io/bnemeth/multiarch-qemu-user-static --reset -p yes
```

Comment:
Is quay.io/bnemeth/multiarch-qemu-user-static going to be permanent?

Reply:
Yes

Reply:
We need some transparency to what this is if this step is going to be used to build production images.

Reply:
If it is only development, this should be good. But I think adding your containerfile in that is used to build multiarch-qemu-user-static in /hack would be good.

Reply:
This is an external repo, I've added a commit message. If need be, we could add it as a submodule, and build and push the container as part of the build process. For now, I suggest we keep the commit message, and use the quay image I've prepared.